pages/blog/event-driven-api-documentation-made-simple-clientside-rendering.md

```diff
@@ -0,0 +1,219 @@
+
+---
+
+### <a name="react"></a>React
+
+If you wish to render documentation from your AsyncAPI file in React application, then you need to use [AsyncAPI React component](https://github.com/asyncapi/asyncapi-react).
+
+1️⃣ To install the **React AsyncAPI component** run the command `npm install --save @asyncapi/react-component@next`.
+
+2️⃣ Now, create **index.js** file and type in:
+
+```
+import React from "react";
+import ReactDOM from "react-dom";
+
+import AsyncApiComponent from "@asyncapi/react-component";
+import "@asyncapi/react-component/styles/default.min.css";
+
+import { specMock } from "./testDoc";
+
+const rootElement = document.getElementById("root");
+ReactDOM.render(<AsyncApiComponent schema={specMock} />, rootElement);
+```
+
+**React AsyncAPI component** is imported on the **line 4**.
+
+If you are happy with AsyncAPI styling then you need to import their CSS pattern with `import "@asyncapi/react-component/styles/default.min.css";`
+
+On the **line 7** is where AsyncAPI sample file is imported.
+
+---
+
+### <a name="vue"></a>Vue
+
+If you wish to generate documentation from your AsyncAPI file in Vue application, then you need to use [AsyncApiStandalone bundle](https://github.com/asyncapi/asyncapi-react/blob/next/docs/usage/vue.md).
+
+1️⃣ React AsyncAPI component is required here as well, so again you need to run `npm install --save @asyncapi/react-component@next` command.
+
+2️⃣ In your **App.vue** just add this code:
+
+```
+<template>
+  <div ref="asyncapi"></div>
+</template>
+
+<script>
+import AsyncApiStandalone from '@asyncapi/react-component/browser/standalone';
+
+// AsyncAPI specification, fetched or pasted.
+const schema =
+'{"asyncapi":"2.4.0","info":{"title":"Account Service","version":"1.0.0","description":"This service is in charge of processing user signups"},"channels":{"user/signedup":{"subscribe":{"message":{"$ref":"#/components/messages/UserSignedUp"}}}},"components":{"messages":{"UserSignedUp":{"payload":{"type":"object","properties":{"displayName":{"type":"string","description":"Name of the user"},"email":{"type":"string","format":"email","description":"Email of the user"}}}}}}}';
+
+const config = {}; // Configuration for component. This same as for normal React component.
+
+export default {
+  name: 'AsyncApiComponent',
+  props: {
+    msg: String
+  },
+  mounted() {
+    const container = this.$refs.asyncapi;
+    AsyncApiStandalone.render({ schema, config }, container);
+  }
+}
+</script>
+
+<style scope src="@/assets/asyncapi.min.css"></style>
+```
+
+As you can see on **line 6**, you need to import **AsyncApiStandalone** bundle with `import AsyncApiStandalone from '@asyncapi/react-component/browser/standalone';` command.
+
+3️⃣ There is one more thing to do. If you like AsyncAPI styling then you need to go to 👉 `node_modules/@asyncapi/react-component/style/default.min.css`.
+
+Copy that file and then paste it into your _**assets**_ folder.
+I renamed mine **asyncapi.min.css**.
```

Comment:
Yeah, it's true, we have similar solution in our docs, but in Vue you can also point to the styles from `node_modules` POV. I will update docs, but we should also point to second approach in this blog post, my suggestion:

```
You can also import styles from `node_modules`:

/```vue
<style scope src='@asyncapi/react-component/style/default.min.css'></style>
/```
```

Please remove `/` before backticks - github markdown preview breaks markdown when you use multiline code blocks in another code blocks.